assets/queries/k8s/rotate_kubelet_server_certificate_not_active/query.rego

```diff
@@ -0,0 +1,54 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.k8s as k8sLib
+
+command = {"kube-apiserver", "kubelet", "kube-controller-manager"}
```

Comment:
I suggest updating the query `Kubelet Server Periodic Certificate Switch Disabled` to `Rotate Kubelet Server Certificate Not Active` instead of creating a new query